source/guide_nextcloud.rst

```diff
@@ -93,19 +191,25 @@ Sendmail Settings
 
 .. warning:: Make sure to replace the example values.
 
-::
+.. code-block:: console
 
- 'mail_domain' => 'uber.space',
- 'mail_from_address' => 'isabell',
- 'mail_smtpmode' => 'sendmail',
- 'mail_sendmailmode' => 'pipe',
+  [isabell@stardust html]$ php occ config:system:set mail_domain --value="uber.space"
+  System config value mail_domain set to string uber.space
+  [isabell@stardust html]$ php occ config:system:set mail_from_address --value="$USER"
+  System config value mail_from_address set to string isabell
+  [isabell@stardust html]$ php occ config:system:set mail_smtpmode --value="sendmail"
+  System config value mail_smtpmode set to string sendmail
+  [isabell@stardust html]$ php occ config:system:set mail_sendmailmode --value="pipe"
+  System config value mail_sendmailmode set to string pipe
 
 SMTP Settings
```

Comment:
Since you're giving this guide an overhaul anyway: Is there a downside to using sendmail? It all ends up in the same mail queue anyway. If there is no downside on the nextcloud side of things, I think having just the easier sendmail option is enough for this guide. Maybe add a `.. note` block below linking to additional docs on alternatives.

Reply:
Users can also do the advanced mail setup (for external servers and stuff) in the web based admin settings (https://isabell.uber.space/settings/admin), so the simple sendmail should be enough for this guide.